packages/create-cloudflare/templates/queues/js/wrangler.toml

```diff
@@ -1,5 +1,5 @@
 name = "<TBD>"
-main = "src/index.js"
+main = "src/index.ts"
```

Comment:
here we are in `/queues/js/` so I think that either the dir name is incorrect/misleading of the extension here is incorrect